android/core/src/main/java/com/pega/vehicledamagemodeling/LimitedCameraInputController.java

```diff
@@ -111,4 +114,41 @@ private boolean isCrossingUpLimit(Vector3 rotateVector, float deltaYRotate) {
         tmpV2.rotate(rotateVector, deltaYRotate);
         return tmpV2.y < ROTATE_UP_LIMIT;
     }
+
+    public void setUpPosition(BoundingBox boundingBox, int width, int height) {
+        float distanceModelToCamera = Math.max(boundingBox.getDepth(), boundingBox.getWidth());
+        float xPositionByPythagorasTriangle = distanceModelToCamera * 12 / 13;
```

Comment:
What are those magic numbers? Where do they come form? Are they model specific? If so, they should be at least extracted as a meaningful constant, that allows for a quick change in one place. 
For example - is this 13 the same as the one below? If I need to change the first one should I also change the second one?